packages/react-composites/src/composites/common/AvatarPersona.tsx

```diff
@@ -46,7 +46,7 @@ export type AvatarPersonaData = {
  *
  * @public
  */
-export type AvatarPersonaDataCallback = (userId: string) => Promise<AvatarPersonaData>;
+export type AvatarPersonaDataCallback = (userId: string) => Promise<AvatarPersonaData> | undefined;
```

Comment:
I think this should be `Promise<AvatarPersonaData | undefined>` instead of `Promise<AvatarPersonaData> | undefined`

i.e. if I'm a developer I can do:
```
onFetchAvatarPersonaData={async (userid) => {
  if (userId === 'soemthing) { return awaitFetchNewVals }
  // else returns undefined by default, which is actually Promise<undefined> as its inside an async function
}
```

Reply:
Changed!